streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -507,7 +544,7 @@ private Topology setupTopologyWithoutIntermediateUserTopic() {
         return builder.build();
     }
 
-    private void cleanGlobal(final boolean withIntermediateTopics,
+    private int tryCleanGlobal(final boolean withIntermediateTopics,
```

Comment:
We could add meta comment for the return value here, and instead of returning an exit code, I feel a boolean is suffice to indicate whether the clean operation was successful or not.

Reply:
Indeed, updated as suggested